scripts/reclassify/reclassify.yml

```diff
@@ -0,0 +1,30 @@
+script: reclassify.R
+name: Reclassifies rasters in collection 
+description: This code reclassifies the pixels of the rasters in a collection into the specified categories.
+author: 
+ - name: Maria Alejandra Molina Berbeo (mmolina@humboldt.org.co)
+ - name: Victor Julio Rincon (vrincon@humboldt.org.co)
+ - name: Luis Fernando Urbina (lurbina@humboldt.org.co)
+external_link: https://github.com/GEO-BON/biab-2.0
```

Comment:
Change ot appropriate link. If there is no external link to put, please remove this.